benchmark/auto/README.md

```diff
@@ -0,0 +1,48 @@
+# AUTO BENCHMARK
+
+Auto benchmark aims to generate testsets with synthetic Q&A pairs from raw passages, using as few human annotations as possible. To achieve this, we use LLM as a annotator and dataset quality inspector.
+
+## Usage
+
+1. Prepare your corpus in a JSON file, following the format
+
+```
+corpus.json: [
+      {"document": "Your document text here"}
+]
+few_shot_cases.json: [
+      {"document": "Sample document", 
+      "Query": "Sample question"}
```

Comment:
In your sample few_shot_cases.json file, it seems that the format is:
{"document": "Sample document", 
      "Query": "Sample question",
"Answer": "Sample answer"}

Reply:
Actually, `Answer` is not used in few-shot cases because I did not employ few-shot cases in answer generation step.  And this is why I did not add `Answer` to the example.